lib/gui/app/components/flash-results/flash-results.tsx

```diff
@@ -199,12 +201,9 @@ export function FlashResults({
 							fontWeight: 500,
 							textAlign: 'center',
 						}}
-						tooltip={outdent({ newline: ' ' })`
-							The speed is calculated by dividing the image size by the flashing time.
-							Disk images with ext partitions flash faster as we are able to skip unused parts.
-						`}
+						tooltip={i18next.t('flash.speedTip')}
 					>
-						Effective speed: {effectiveSpeed} MB/s
+						{i18next.t('flash.speed')} {effectiveSpeed} MB/s
```

Comment:
The ` MB/s` should be part of the localisable string (with the `effectiveSpeed` passed in as a parameter), rather than just being tacked on the end.

Reply:
Finished, thanks!